### PR TITLE
docs: update qase-cucumberjs readme

### PR DESCRIPTION
**Pull request by gibiw**
docs: update qase-cucumberjs readme
--
Update qase-cucumberjs readme:
- add section how to update from v1
- update examples
- update execution command
- update configuration section

### CODE REVIEW

File: qase-cucumberjs/README.md [hunk 1]
```diff
@@ -1,50 +1,70 @@
...
-> Publish results simple and easy.
+# Qase TMS Cucumber JS reporter
 
-## How to integrate
+Publish results simple and easy.
 
+## How to install
+
+```sh
+npm install -D cucumberjs-qase-reporter
```
Comment (NickVolynkin) [inline qase-cucumberjs/README.md:8]: until it's released
```suggestion
npm install -D cucumberjs-qase-reporter@beta
```

File: qase-cucumberjs/README.md [hunk 2]
```diff
@@ -1,50 +1,70 @@
...
 ```
-npm install cucumberjs-qase-reporter
-```
 
-## Example of usage
+## Updating from v1
+
+You can update a test project from using version 1 to version 2 in several steps:
+
+1. Update reporter configuration in `qase.config.json` and/or environment variables —
```
Comment (NickVolynkin) [inline qase-cucumberjs/README.md:15]: Just one step? Didn't import paths change?

File: qase-cucumberjs/README.md [hunk 3]
```diff
@@ -1,50 +1,70 @@
...
+```sh
+npm install -D cucumberjs-qase-reporter
 ```
-npm install cucumberjs-qase-reporter
-```
 
-## Example of usage
+## Updating from v1
+
+You can update a test project from using version 1 to version 2 in several steps:
```
Comment (NickVolynkin) [inline qase-cucumberjs/README.md:13]:
```suggestion
To update a test project using cucumberjs-qase-reporter@v1 to version 2:
```

File: qase-cucumberjs/README.md (hunk 1 above)
Comment (gibiw) [reply qase-cucumberjs/README.md]: fixed

File: qase-cucumberjs/README.md (hunk 2 above)
Comment (gibiw) [reply qase-cucumberjs/README.md:15]: yep, only one step

File: qase-cucumberjs/README.md (hunk 3 above)
Comment (gibiw) [reply qase-cucumberjs/README.md]: fixed

File: qase-cucumberjs/README.md [hunk 4]
```diff
@@ -1,50 +1,70 @@
...
+  As a user of cucumber.js
+  I want to have documentation on cucumber
+  So I can write better applications
+
+  @QaseID=1
+  Scenario: Usage documentation
+    Given I am on the cucumber.js GitHub repository
+    When I go to the README file
+    Then I should see a "Cool" section
+    When I go to the README file
```
[the remark this answers is not in the document]
Comment (gibiw) [reply qase-cucumberjs/README.md]: fixed

File: qase-cucumberjs/README.md [hunk 5]
```diff
@@ -105,41 +119,17 @@ Supported ENV variables:
...
-      './tests/e2e/specs/*.js',
-    ],  // require step definition files before executing features
-    tags: [],
-    'dry-run': false,
-    compiler: [],  
-    format: ["node_modules/cucumberjs-qase-reporter"],
-  },
-  ...
-}
+cucumber-js -f cucumberjs-qase-reporter features
```
[the remark this answers is not in the document]
Comment (gibiw) [reply qase-cucumberjs/README.md]: fixed

**Review by NickVolynkin — APPROVED** [review]

**Pull request merged**